client/settings/payment-method-settings/digital-wallets-settings.js

```diff
@@ -2,13 +2,84 @@
 /**
  * External dependencies
  */
-import { Card, CardBody } from '@wordpress/components';
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button, Card, CardBody, RadioControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
 
 const DigitalWalletsSettings = () => {
+	const [ cta, setCta ] = useState( 'buy' );
+	const [ size, setSize ] = useState( 'default' );
+	const [ theme, setTheme ] = useState( 'dark' );
+
 	return (
-		<Card>
-			<CardBody>Digital Wallets placeholder.</CardBody>
-		</Card>
+		<>
+			<Card>
+				<CardBody>
+					<h4>{ __( 'Call to action', 'woocommerce-payments' ) }</h4>
+					<RadioControl
+						help="Select a button label that fits best with the flow of purchase or payment experience on your store."
```

Comment:
This string should go under translation
```suggestion
						help={ __( 'Select a button label that fits best with the flow of purchase or payment experience on your store.', 'woocommerce-payments' ) }
```

Reply:
D'oh! Good catch, I've added the i18n call to all the help and option labels now.